.github/workflows/workflow-run-replay-verify.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
     runs-on: ${{ inputs.RUNS_ON }}
     strategy:
         matrix:
-          number: [0, 1, 2, 3, 4, 5, 6, 7] # runner number
+          number: [0, 1] # runner number
```

Comment:
Is there a way to make this more configurable?

Reply:
there should be. But I am not an expert on github config and didn't figured out